### PR TITLE
Add page title on all pages

### LINKED ISSUE #178: Update Title of page
Change the page `<title></title>` to the following when viewing content.

`<section name> : <content name>`

Possible ways of implmenting:

- Implement a new `title` service, it will appropriate methods to set/get title, section, content name etc..
- You will need to Jquery to update the title, since it is outside ember app.

- Or you can use the addon: http://tim-evans.github.io/ember-page-title/

EDIT: Also update title suitably for all the pages.

### PR DESCRIPTION
**Pull request by architkshk**
Using [ember-cli-document-title](https://github.com/kimroen/ember-cli-document-title) . Fixes #178 .
@abhishek97

### CODE REVIEW

**Comment by abhishek97** [comment]
@architkshk can you test it now?

**Comment by architkshk** [comment]
@abhishek97 Done!! 
screenshot from 2018-07-03 13-24-08

**Comment by abhishek97** [comment]
@architkshk The title must also change when user redirects back to home page(or any other page for that matter)

**Comment by architkshk** [comment]
@abhishek97  Done! set up application title to use when no title is defined for the route.

**Comment by architkshk** [comment]
@abhishek97 Added titles for all pages

**Comment by architkshk** [comment]
While loading - **CodingBlocks Online**,
Home - **Home | CodingBlocks Online**,
My Courses  - **My Courses | CodingBlocks Online**,
All Courses  - **All Courses | CodingBlocks Online**,
Classroom course (My Course) -  **_Overview/Doubts/Contents/Announcements_ - _Course Name_ | CodingBlocks Online**,
Courses course (All Courses) -  **_Course Name_ | CodingBlocks Online**,

**Comment by architkshk** [comment]
@abhishek97 
screenshot from 2018-07-03 18-12-20
screenshot from 2018-07-03 18-12-35

File: app/pods/attempt/content/index/route.js [hunk 1]
```diff
@@ -15,6 +15,12 @@ export default Route.extend({
         controller.set("content", model.content)
         controller.set("payload", model.payload)
     },
+    titleToken: function(model){
+        let contentName = model.payload.get('name');
+        let sectionName = model.runAttempt.get('run').get('sections')
```
Comment (abhishek97) [inline app/pods/attempt/content/index/route.js]: if you have `sectionId` just get it directly from the store. no need to `find`

**Comment by architkshk** [comment]
@abhishek97  made the required changes

**Pull request closed without merging**